src/main/java/Duke.java

```diff
@@ -1,10 +1,62 @@
+import java.util.Scanner;
+import java.util.ArrayList;
 public class Duke {
+    public static void printHorizontalLine() {
+        System.out.println("____________________________________________________________");
+    }
+
+    public static void printHelloMessage(String logo) {
+        System.out.println("Hello from\n" + logo);
+        printHorizontalLine();
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?");
+        printHorizontalLine();
+    }
+
+    public static void checkInput(String input, ArrayList<Task> listOfTasks) {
+        String[] inputs = input.split(" ");
+        switch (inputs[0]) {
+        case "bye":
+            System.out.println("Bye. Hope to see you again soon!");
+            break;
+        case "list":
+            Task.listTasks(listOfTasks);
+            break;
+        case "mark":
+            Task markTask = listOfTasks.get(Integer.parseInt(inputs[1])-1);
```

Comment:
Perhaps you could name "markTask" differently as a noun? The current naming makes it sound like a verb (mark the task) which is usually used to name methods.

Reply:
thank you